### PR TITLE
Add instructions to clone repo for examples

### PR DESCRIPTION
**Pull request by MadhavJivrajani**
Added instructions for cloning the repository in the examples section of the docs as per #3040

### CODE REVIEW

**Review by nrb — DISMISSED** [review]
Thanks for this contribution!

Normally, we don't need the docs changes for versions outside of `main` or the latest (`v1.5` in this case).

Since the examples _may_ be tied to a given release, could you please remove the changes in the v1.3.x and v1.4 directories?

**Comment by MadhavJivrajani** [comment]
> Thanks for this contribution!
> 
> Normally, we don't need the docs changes for versions outside of `main` or the latest (`v1.5` in this case).
> 
> Since the examples _may_ be tied to a given release, could you please remove the changes in the v1.3.x and v1.4 directories?

Hi,
I've made the changes

File: site/content/docs/v1.2.0/examples.md [hunk 1]
```diff
@@ -3,7 +3,11 @@ title: "Examples"
 layout: docs
 ---
 
-After you set up the Velero server, try these examples:
+After you set up the Velero server, you can clone the examples used in the following sections by running the following:
```
Comment (ashish-amarnath) [inline site/content/docs/v1.2.0/examples.md:6]: Change to this file can also be reverted.

**Review by carlisia — APPROVED** [review]

**Review by ashish-amarnath — APPROVED** [review]

**Comment by ashish-amarnath** [comment]
@nrb this is PR is good for merge. PTAL.

**Pull request merged**